src/main.rs

```diff
@@ -130,12 +133,17 @@ fn build_ui(app: &Application) -> Result<()> {
 
     if let Some(base_url) = qqmusic_api_base_url {
         let base_url = url::Url::from_str(&base_url)?;
-        QQMUSIC_API_CLIENT.set(Some(QQMusicApi::new(base_url)));
+        let _ = QQMUSIC_API_CLIENT.set(Some(QQMusicApi::new(base_url)));
```

Comment:
Is `let _ = ` necessary here?

Reply:
No, it's here to mute compiler warning about unused result, but that's what we expect